src/index.html

```diff
@@ -7,21 +7,396 @@
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
     />
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400&display=swap"
       rel="stylesheet"
-      href="styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header_logo"
+        href="index.html"
+      >
+        <img
+          class="header_logo_img"
+          src="./images/logo.png"
+          alt="Moyo_logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#index.html"
+            >
+              APPLE
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+            >
+              SAMSUNG
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+            >
+              TABLETS
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+            >
+              PHOTO
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#index.html"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content-main">
+      <section class="catalog">
+        <div
+          class="product-card"
+          data-qa="card"
+        >
+          <div class="product-card__image"></div>
+          <div class="product-card__content">
+            <p class="product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </p>
+            <div class="product-code">Product code: 195434</div>
+            <div class="product-rating">
+              <div class="stars">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
```

Comment:
The stars rating system should be implemented with CSS classes. The 'stars' block should have a modifier class 'stars--x' where 'x' is a number from 1 to 5. This class should change the appearance of the stars to represent the rating. You should not use empty div elements for this, it's better to use CSS pseudo-elements or background images.